crates/wasmtime/src/module/serialization.rs

```diff
@@ -741,6 +741,16 @@ impl<'a> SerializedModule<'a> {
 
         Ok(())
     }
+
+    pub fn mlock(&self) -> Result<()> {
```

Comment:
With my above suggestion of moving to `Module::from_parts` I think that means this could probably be removed